index.js

```diff
@@ -0,0 +1,24 @@
+const { MongoClient } = require('mongodb')
+const express = require('express')
+const bodyParser = require('body-parser')
+const app = express()
+
+MongoClient.connect('mongodb://localhost/library', function(err, db) {
+  app.use(express.static('public'))
+  app.use(bodyParser.json())
+
+  app.post('http://ergast.com/api/f1/2010/drivers.json', (req, res) => {
```

Comment:
If your client does not send any `POST` requests yet, then you can remove this route handler.